resources/views/clans.blade.php

```diff
@@ -55,12 +55,12 @@ class="fas fa-sort"></i>
                                     <tr class="data_row">
                                         <th scope="row">{{ $clan['rank'] }}</th>
                                         <td>{!! $clan['color_tag'] !!}</td>
-                                        <td>{{ $clan['name'] }}</td>
+                                        <td style="cursor: pointer" onclick="showClan('{{$clan["tag"]}}');">{{ $clan['name'] }}</td>
```

Comment:
That's a bad practice to use *onclick*. The main reasons:
* it mixes code and markup
* the code written in this way passes through [eval](https://javascript.info/eval#using-eval)

Pluses of usage separate .js files:
* You can add behavior to multiple HTML elements at once without duplicating the code.
<hr>
It's okay here, but we should keep in mind just in case.

Reply:
I don't know how we could handle that in separate js files. We need the ID (player or name).

Reply:
> 
> 
> I don't know how we could handle that in separate js files. We need the ID (player or name).

document.querySelector(".data_row>.home").innerHTML ...